### ISSUE TITLE
Decomposer NEI recipe lookup bug caused by Mekanism and Galacticraft Planets

### ISSUE THREAD

**Issue by ChatFawkes** [jakimfett/Minechem#670]
Minechem Version: 5.0.5.387

Crash occurs when attempting to use recipe lookup button for Chemical Decomposer

Crash Report: http://pastebin.com/4pWD0ML0

Only occurs when Mekanism and/or Galacticraft are installed.

I have removed every mod and added them back one by one (and eventually groups at a time) to find out that Mekanism and Galacticraft seem to be causing this issue with the NEI lookup for the Decomposer.

Edit: my guess is because these two mods add gases.

Edit2: Possibly related to #626, but his crash report didn't have either Mekanism or Galacticraft.

**Comment by ChatFawkes** [comment]
Alright I've just confirmed this happens with only:
CoFHCore, Minechem, CodeChickenCore + Lib, NotEnoughItems, Mekanism + FMP.

Also it happens with the following mods:
CoFHCore, Minechem, CodeChickenCore + Lib, NotEnoughItems, and Galacticraft Core + Planets.
However with the Galacticraft it only happens when you have Planets installed. If you only have GalacticraftCore it works just fine.

And obviously it happens with both Mekanism + Galacticraft Core + Planets installed.

I've noticed that Galacticraft Planets adds Liquid Nitrogen, Liquid Oxygen, and Methane Gas.
Mekanism also adds Oxygen, so perhaps that is related?

Hope this helps!
